tests/test_commands.py

```diff
@@ -0,0 +1,15 @@
+#
+# Copyright (C) 2021 FreeIPA Contributors see COPYING for license
+#
+
+import os
+
+from ipapython.ipautil import run
+
+
+def test_version():
+    """
+    Test the --version option
+    """
+    output = run(['ipa-healthcheck', '--version'], env=os.environ)
```

Comment:
no need to pass unmodified `env`, will be copied from current by default.

Reply:
ipautil.run clears PATH if environment isn't passed in which breaks a virtual environment.

Reply:
Not obvious behavior for me, but OK.